net/lp_rpc.pb.go

```diff
@@ -1938,6 +1999,7 @@ func init() {
 	proto.RegisterFile("net/lp_rpc.proto", fileDescriptor_034e29c79f9ba827)
 }
 
+// TODO: How do I regenerate this?
```

Comment:
Could anyone help me to regenerate this bit? Is this whole file actually auto-generated? How can I trigger it myself?

Reply:
I regenerated the file in 1dd38ec8cb3176cfa5368f67a2a4195bf2797504

FYI: to regenerate the file I did execute the following command:
```
protoc --go_out=plugins=grpc:. net/lp_rpc.proto
```

I think it should be executed in `Makefile`, but for some reason it does not work 🤔 I think it's a quick fix, but... we also need to fix the `protoc` version. Currently it only works with the specific version. Please check this [Discord discussion](https://discord.com/channels/423160867534929930/943138984698200137/943521382564769862).

Anyway, at some point, it's worth spending some time to update protoc to the latest version + fix Makefile.

Reply:
Great, thanks!

Yesterday I did actually find that makefile command, tried installing protoc but then it generated some files very different from the current ones, so I didn't know which version to install instead. If protoc is installable via go, I think there's a way to install a "tool" and then invoke a tool in the specific version installed in the project. We could try doing that later.

Regarding makefile, the issue was that the build step is not phony. By default, if a file exists with the same name of a step, it assumes that it has already been made and skips the build step. Since the make step has exactly the name of the go file that is generated it doesn't regenerate it.

To skip that check we can make the build step phony, like some others that we have already in the makefile with something like:
`.PHONY net/lp_rpc.pb.go`

Then it will run the step even if the file already exists.

Reply:
Yes, but TBH I'd do it in a separate PR. And make sure everyone who runs `make` (everyone!) uses the same protoc golang plugin version. Otherwise, it'll be a burden, because you run make and the generated file is suddenly completely different (the same what you experienced).